### PR TITLE
3.1 release notes

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| Versions      | master |

A preliminary version of the RN article with URL wildcards description. Certain key sections commented out.

### CODE REVIEW

File: docs/releases/ez_platform_v3.1.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Notable changes
+
+### URL management UI
+
+You can now manage URL addresses and URL wildcards with a comfortable user interface that is available in the Back Office. You can create, modify and delete URL wildcards, as well as decide if the user should be redirected to the new address on clicking the link.
+
+!!! note
+
+As of this release, the Link manager is no longer part of the Content panel, and now it belongs to the Admin panel of the Back Office.
```
Comment (dabrt) [inline docs/releases/ez_platform_v3.1.md:19]:
```suggestion
  As of this release, the Link manager is no longer part of the Content panel, and now it belongs to the Admin panel of the Back Office.
```

File: docs/releases/ez_platform_v3.1.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+For more information about configuring the Universal Discovery Widget, see [URL management](../extending/extending_udw.md) in the developer documentation.
+
+### Saving incomplete draft
+
+As of this release, when users create or edit a Content item or a Page, they can save it without completing all the required fields.
+They can then return to editing, or pass the content to another contributor.
+Validation that used to happen at each save operation now, by default, happens when you click the **Publish** button.
+
+Also, as part of this feature, the `ContentService::validate()` method has been added that you can use to trigger validation of individual fields or whole Content items for completeness at other stages of the workflow.
```
Comment (dabrt) [inline docs/releases/ez_platform_v3.1.md:45]:
```suggestion
Also, as part of this feature, the `ContentService::validate()` method has been added that you can use to trigger validation of individual fields 
or whole Content items for completeness at other stages of the editing process.
```

File: docs/releases/ez_platform_v3.1.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+### PermissionResolver
+
+You can now have a Service that provides both `PermissionResolver` and `PermissionCriterionResolver` by injecting `eZ\Publish\API\Repository\PermissionService`.
+
+## Behavior changes
+
+### Landing Page drafts
+
+When you start editing a Landing Page, a new draft is now automatically created.
```
Comment (SylvainGuittard) [inline docs/releases/ez_platform_v3.1.md:117]: `editing` or `creating`?

Comment (DominikaK) [inline docs/releases/ez_platform_v3.1.md:117]:
```suggestion
When you start creating a Landing Page, a new draft is now automatically created.
```

File: docs/releases/ez_platform_v3.1.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+![Field group tabls in Content editing](img/3.1_collapsible_fields_edit.png)
+
+In Content preview, the group sections are collapsible:
+
+![Collapsible Field groups in Content view](img/3.1_collapsible_fields.png)
+
+### Saving incomplete draft
+
+As of this release, when users create or edit a Content item or a Page, they can save it without completing all the required fields.
```
Comment (DominikaK) [inline docs/releases/ez_platform_v3.1.md:102]:
```suggestion
When users create or edit a Content item or a Page, they can now save it without completing all the required fields.
```

File: docs/releases/ez_platform_v3.1.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+They can then return to editing, or pass the content to another contributor.
+Validation that used to happen at each save operation now, by default, happens when you click the **Publish** button.
+
+Also, as part of this feature, the `ContentService::validate()` method has been added that you can use to trigger validation of individual fields
```
Comment (DominikaK) [inline docs/releases/ez_platform_v3.1.md:106]:
```suggestion
The `ContentService::validate()` method has been added that you can use to trigger validation of individual fields 
```

File: docs/releases/ez_platform_v3.1.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+Also, as part of this feature, the `ContentService::validate()` method has been added that you can use to trigger validation of individual fields 
+or whole Content items for completeness at other stages of the editing process.
+
+### Search
+
+#### ezplatform-search
+
+[`ezplatform-search`](https://github.com/ezsystems/ezplatform-search) is a new repository
+that contains search functionalities that are not dependent on the search engine
```
Comment (dabrt) [inline docs/releases/ez_platform_v3.1.md:114]:
```suggestion
that contains search functionalities that are not dependent on the search engine.
```

**Pull request merged**